bciers/apps/administration/app/components/operators/OperatorDetailsPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { UUID } from "crypto";
+import { validate as isValidUUID } from "uuid";
+import Loading from "@bciers/components/loading/SkeletonForm";
+import { Suspense } from "react";
+import getOperator from "@/administration/app/components/operators/getOperator";
+import { fetchDashboardData } from "@bciers/actions";
+import { OperatorRow } from "@/administration/app/components/operators/types";
+import { ContentItem } from "@bciers/types/tiles";
+import Tiles from "@bciers/components/navigation/Tiles";
+import updateDashboardDataHref from "@bciers/utils/updateDashboardDataHref";
+
+const OperatorDetailsPage = async ({ operatorId }: { operatorId: UUID }) => {
+  let operator: OperatorRow | { error: string };
+  let operatorDashboardData: ContentItem[] = [];
+
+  if (operatorId && isValidUUID(operatorId)) {
+    operator = await getOperator(operatorId);
+    if (operator && "error" in operator)
+      throw new Error("Failed to retrieve Operator details");
+  } else throw new Error(`Invalid operator id: ${operatorId}`);
```

Comment:
This is the error being thrown on New Operator 7 Legal Name

Reply:
Thank you for noticing this! It wasn't just Operator 7; quite few other operators had incorrect UUIDs.